resources/views/auth/login.blade.php

```diff
@@ -84,7 +84,7 @@
 <div class="col-md-3"></div>
 <script type="text/javascript">
 $(document).ready(function() {
-    $('#login-form').submit(function (event) {
+    $('#login-form').one("submit", function (event) {
```

Comment:
Please use single quotes for all Javascript stuff.

This should be `.one('submit', function (event) {`